google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/standard/AppEngineStandardFrameworkDetector.java

```diff
@@ -39,7 +39,7 @@
     FacetBasedFrameworkDetector<AppEngineStandardFacet, AppEngineStandardFacetConfiguration> {
 
   public AppEngineStandardFrameworkDetector() {
-    super("appengine-java");
+    super("appengine-java-standard");
```

Comment:
Since you changed this here, do you also need to change it in `AppEngineStandardFrameworkType`?